.build-test-rules.yml

```diff
@@ -84,3 +84,24 @@ components/icm42670:
 components/qma6100p:
   depends_filepatterns:
     - "components/qma6100p/**"
+
+components/io_expander/esp_io_expander_ht8574:
+  depends_filepatterns:
+    - "components/io_expander/esp_io_expander_ht8574/**"
```

Comment:
The component also depends on `components/io_expander/esp_io_expander`. Could you please add it to all 3 test_apps?